src/main/app/controller/courts/AddressController.ts

```diff
@@ -400,6 +421,15 @@ export class AddressController {
     return errors;
   }
 
+  private validateEPIMId(courtAddress: DisplayAddress): string[] {
```

Comment:
You could do this in one line if you wanted to:

courtAddress.epim_id && !/^[a-zA-Z0-9-]{0,30}$/.test(courtAddress.epim_id.trim()) ? [this.invalidEpimError] : []; 

Or something like that

Reply:
Since you are only pushing one error as well, do we need a list? Could simply return the invalidEpimError. Up to you on this one, as if other requirements come in it may make it applicable to have a list I suppose